### DISCUSSION TITLE
Add to Carousel new Embla options: `watchDrag`, `watchResize`, `watchSlides` from release `v8.0.0`

### DISCUSSION THREAD

**Discussion by eugeniudedin** [mantinedev/mantine#6295] [Feature requests]
### watchDrag
This new option adds the possibility to cancel the default drag behaviour. For example, it's useful if you need to disable dragging when a user wants to scroll an element inside a slide. 

### watchResize
Embla is no longer listening to the window resize event. Instead, it's using `ResizeObserver` on its container and slides. If any size has changed it will re-initialize by calling `reInit` automatically.

### watchSlides
This new option adds the feature of Embla automatically re-initializing by calling `reInit` when slides have been added or removed, using `MutationObserver`.

**Comment by rtivital** [comment]
Upgrading embla version requires breaking changes. No changes are planned to be made for carousel package until 8.00 release.

**Reply by eugeniudedin** [reply]
Hey Vitaly 👋 Thanks for the answer. All the best!

**Reply by jamalsoueidan** [reply]
Direction is also missing.
